src/site/topic22.rst

```diff
@@ -202,11 +202,53 @@ Traversal Analysis
 
 * :math:`O(2^{h})`
 * Why?
-* Consider the relationship between the height of a binary tree and the number of nodes within the tree  
+* Consider the relationship between the height of a binary tree and the number of nodes within the tree
+
 
 Interface
 =========
 
+* What would we want to do with our binary trees?
+    * Check if an element exists in the tree
+    * Check if the tree is empty
+    * Get the size of the tree (how many elements are in the tree)
+    * Traverse the tree
+    * Add an element
+        * But where?
+    * Remove an element
+        * Which one? From where?
```

Comment:
This is kind of the point of abstraction in general. The point of `add` is to add something, I don't really care about how that takes place. Take [Collection#add](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Collection.html#add(E)) as an example, a collection is more generic than a binary tree even, yet it still defines that stuff can be added (and removed) from a collection.

Further, consider how'd you use this in practice. Take this snippet:
```java
BinaryTree<Integer> tree = new RedBlackTree<Integer>();
tree.add(10);  // will not compile, no `add` exists
```
I actually have to remove the abstraction to be able to use the binary tree.

To sum it up,
> add an element, but where?  

_I don't know, I don't want to know_.

Reply:
I went back through #199 to see why exactly they were excluded and came to no conclusion. I have a faint memory of there being a reason, but can't think of one at all. 

I"m going to fix this and add the add and remove to the interface.